docs/spec_draft.md

```diff
@@ -1526,6 +1527,64 @@ operations correspond to [stablehlo.minimum](#stablehlominimum) and
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.collective_permute
+
+### Semantics
+
+Within each process group in the StableHLO grid, sends and receives data between
+processes in the group.
+
+The operation splits the StableHLO grid into `process_groups` as follows:
```

Comment:
The operation splits the StableHLO grid into `source_target_pairs ` as follows:
* `channel_id <= 0`,
    `source_target_pairs` = `cross_replica(replica_groups)`.
* `channel_id > 0`,
    `source_target_pairs` = `cross_replica(replica_groups)`.

Reply:
We are consistently using `process_groups` to denote the result from various group forming strategy. [example](https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md#semantics-3)

Reply:
sounds fair